pkg/apiserver/domain/service/rbac.go

```diff
@@ -234,6 +234,17 @@ var ResourceMaps = map[string]resourceMetadata{
 				pathName: "configName",
 			},
 			"provider": {},
+			"pipeline": {
```

Comment:
Set the default role and permissions for the Pipeline resource.

Reply:
done